Template/LO/SubProcesses/unwgt.f

```diff
@@ -35,6 +35,7 @@ subroutine Zoom_Event(wgt,p)
       double precision xzoomfact
       common/to_zoom/  xzoomfact
       include 'run.inc'
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc